monorepo_prep.sh

```diff
@@ -87,30 +88,33 @@ function patch_catalogd_makefile() {
  IMAGE_REPO := quay.io/operator-framework/catalogd
 EOF
 
-    patch -p1 catalogd/Makefile < makefile.patch
+    patch -p1 < makefile.patch
 
-    if [ -f ./makefile.patch ]
-    then
-        rm -fv makefile.patch
-    fi
+    rm -f makefile.patch
 
     git add catalogd/Makefile
-
     git commit -s -m "Update go.mod location in catalogd/Makefile"
 }
 
+function test_binaries_build() {
+    echo "Testing binaries build..."
+    make build-linux || exit 1
+    cd catalogd || exit 1
+    make build-linux || exit 1
+    cd - || exit 1
+}
+
+function check_in_generated_files() {
+    echo "Checking in generated files..."
+    git add --all
+    git commit -s -m "Check in generated manifest files"
+}
+
+# Execute the steps
 catalogd_prep
-operator-controller_prep
+operator_controller_prep
```

Comment:
Thanks for making the name consistent, hyphen vs underscore

Reply:
No... this is not correct... the propername is "operator-controller" and the _prep is the next token in the function name.
There is no confusion this is on purpose